bigtable/client/internal/table_admin.h

```diff
@@ -68,6 +68,35 @@ class TableAdmin {
         rpc_backoff_policy_(backoff_policy.clone()),
         metadata_update_policy_(instance_name(), MetadataParamTypes::PARENT) {}
 
+  /**
+   * Create a new TableAdmin using explicit copy constructor implementation.
+   */
+  TableAdmin(const TableAdmin& table_admin)
+      : client_(std::move(table_admin.client_)),
+        instance_id_(std::move(table_admin.instance_id_)),
+        instance_name_(table_admin.InstanceName()),
+        rpc_retry_policy_(table_admin.rpc_retry_policy_.get()->clone()),
```

Comment:
Alternatively, you could replace the `std::unique_ptr<>` objects with `std::shared_ptr<>` objects and the compiler will generate all the right copy and move operators for you.